source/particle/interpolator/biquadratic_least_squares.cc

```diff
@@ -148,6 +156,14 @@ namespace aspect
 
                 Point<dim> support_point = *itr;
                 cell_properties2[i] = c(0,0) + c(1,0)*(support_point[0]) + c(2,0)*(support_point[1]) + c(3,0)*(support_point[0] * support_point[1]) +  c(4,0)*(support_point[0] * support_point[0]) + c(5,0)*(support_point[1] * support_point[1]);
+                if (cell_properties2[i]>max_value_for_particle_property)
```

Comment:
@egpuckett , @yinghe616 
Correct me if I am wrong, but this method will not work to reduce overshoot and undershoot for the bi-quadratic case. @jonny97, can you please uncommit the changes to biquadratic_least_squares.cc??

```
                if (cell_properties2[i]>max_value_for_particle_property)
                 {
                     cell_properties2[i]=max_value_for_particle_property;
                 }
                 if (cell_properties2[i]<min_value_for_particle_property)
                 {
                       cell_properties2[i]=min_value_for_particle_property;
                 }
```